Proiect/src/Persoane/Angajat.java

```diff
@@ -0,0 +1,33 @@
+package Persoane;
+
+public class Angajat extends Persoana{
+    private String NumeJob;
```

Comment:
[Fix] in java se foloseste camelCase. folosim PascalCase doar pt numele de clase